lib/tasks/update_manual_organisation.rake

```diff
@@ -21,12 +21,15 @@ task :update_manual_organisation, %i[manual_base_path organisation_slug] => :env
   # Update the record in the local database to allow the members of the
   # given organisation to access the manual in Manuals Publisher
   logger.info "Updating record in local database"
-  ManualRecord.find_by(manual_id:).update!(organisation_slug:)
+  manual_record = ManualRecord.find_by(manual_id:)
+  manual_record.update!(organisation_slug:)
+  section_content_ids = m.latest_edition.section_uuids
 
   # Use the Publishing API to update the document
   # across GOV.UK and front end applications
   logger.info "Updating organisations links in Publishing API"
-  Services.publishing_api.patch_links(manual_id, links: { organisations: [organisation_id] })
+  Services.publishing_api.patch_links(manual_id, links: { organisations: [organisation_id], primary_publishing_organisation: [organisation_id] })
+  section_content_ids.each { |section_id| Services.publishing_api.patch_links(section_id, links: { organisations: [organisation_id], primary_publishing_organisation: [organisation_id] }) }
```

Comment:
From a "DRY" point of view, this might be more succinct:

```suggestion
  [manual_id, *section_content_ids].each do |content_id|
    Services.publishing_api.patch_links(content_id, links: { organisations: [organisation_id], primary_publishing_organisation: [organisation_id] })
  end
```